### PR TITLE
Issue2 - Fixed problem with XML header and added test

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

### CODE REVIEW

File: src/main/java/com/redhat/fuse/apicurio/jaxrs/GenerateFuseProjectResource.java [hunk 1]
```diff
@@ -151,9 +151,8 @@ public String generateCamelContextXML(HashMap<String, Object> variables) throws
 
         // Trim off the root elements of the generated XML
         restXML = restXML.trim();
-        restXML = restXML.replaceFirst(Pattern.quote(
-                "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n" +
-                        "<rests xmlns=\"http://camel.apache.org/schema/spring\">\n"), "");
+        restXML = restXML.replaceFirst("\\<\\?xml(.+?)\\?\\>", "").trim();
```
Comment (apupier) [inline src/main/java/com/redhat/fuse/apicurio/jaxrs/GenerateFuseProjectResource.java]: so much more readable than the Pattern quote 👍

File: src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	 * Rest DSL configuration and return it as a string. 
+	 * 
+	 * @param filePath
+	 * @return
+	 * @throws Exception
+	 */
+	private String getRestDSLXMLFromSwagger(String filePath) throws Exception {
+        GenerateFuseProjectResource generator = new GenerateFuseProjectResource();
+        File swaggerFile = new File(filePath);
+        Assert.assertTrue(swaggerFile.exists());
```
[the remark this answers is not in the document]
Comment (bfitzpat) [reply src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: static import? got an example?

[the remark this answers is not in the document]
Comment (bfitzpat) [reply src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: may add assertJ in another round as we add more tests, but for now we'll keep it simple

File: src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	 * create the Rest DSL configuration and return it as a string.
+	 * 
+	 * @param filePath
+	 * @return
+	 * @throws Exception
+	 */
+	private String getRestDSLXMLFromSwagger(String filePath) throws Exception {
+		GenerateFuseProjectResource generator = new GenerateFuseProjectResource();
+		File swaggerFile = new File(filePath);
+		Assert.assertTrue(swaggerFile.exists());
```
Comment (bfitzpat) [inline src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: @apupier - Though the assertj idea is a good one, we'll stick with this for now and shift in a separate PR as we add more tests. But I'm not sure what you meant by handling this through a static import? Do you have an example?

Comment (apupier) [reply src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html

File: src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (apupier) [reply src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html

[the remark this answers is not in the document]
Comment (bfitzpat) [reply src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java]: Added static imports for assertEquals and assertTrue

**Pull request merged**